### PR TITLE
Increasing wait time for standby TOR to go to unhealthy state on Cisco devices

### PR DESCRIPTION
**Pull request by svelamal**
### Description of PR

Summary:
Fixes # (issue)

### Type of change
- [ ] Bug fix

### Back port request
- [ ] 202205

#### What is the motivation for this PR?

1. The testcase marker is t0 and followed by a testbed marker to run it only on dualtor testbeds. The redundancy has been removed by adding dualtor marker
2. Logs collected during the test run indicate that the Ethernet72 (which was chosen randomly) on lower tor stays in active state after 20 secs and becomes standby, unhealthy only after ~25 seconds pass. 

22/03/2023 22:18:47 utilities.wait_until                     L0109 DEBUG  | Time elapsed: **21.907208** seconds
22/03/2023 22:18:47 base._run                                L0063 DEBUG  | /data/tests/common/devices/multi_asic.py::_run_on_asics#108: [m64-tor-1] AnsibleModule::shell, args=["show muxcable status --json"], kwargs={}
  \"Ethernet72\": {\n            \"STATUS\": \"active\",\n            \"SERVER_STATUS\": \"active\",\n            \"HEALTH\": \"unhealthy\"

22/03/2023 22:18:49 utilities.wait_until                     L0109 DEBUG  | Time elapsed: **24.659890** seconds
22/03/2023 22:18:49 base._run                                L0063 DEBUG  | /data/tests/common/devices/multi_asic.py::_run_on_asics#108: [m64-tor-1] AnsibleModule::shell, args=["show muxcable status --json"], kwargs={}
  \"Ethernet72\": {\n            \"STATUS\": \"standby\",\n            \"SERVER_STATUS\": \"standby\",\n            \"HEALTH\": \"unhealthy

#### How did you do it?
Increased the wait time to check the lower tor for standby active state bu 10 seconds and the testcase passes

### CODE REVIEW

File: tests/dualtor/test_server_failure.py [hunk 1]
```diff
@@ -1,23 +1,19 @@
...
-from tests.common.helpers.assertions import pytest_assert, pytest_require
+from tests.common.helpers.assertions import pytest_assert
 from tests.common.dualtor.dual_tor_utils import show_muxcable_status, rand_selected_interface           # noqa: F401
 from tests.common.fixtures.ptfhost_utils import change_mac_addresses, run_garp_service, \
                                                 run_icmp_responder                                      # noqa: F401
 from tests.common.utilities import wait_until
 
 pytestmark = [
-    pytest.mark.topology('t0'),
+    pytest.mark.topology('dualtor'),
```
Comment (wangxin) [inline tests/dualtor/test_server_failure.py:12]: Topology `dualtor` is a type of t0 topology. This mark should not be changed.

File: tests/dualtor/test_server_failure.py [hunk 2]
```diff
@@ -1,23 +1,19 @@
...
     pytest.mark.usefixtures('run_garp_service', 'run_icmp_responder')
 ]
 
 
-@pytest.fixture(autouse=True, scope='module')
-def skip_if_non_dualtor_topo(tbinfo):
-    pytest_require('dualtor' in tbinfo['topo']['name'], "Only run on dualtor testbed")
```
Comment (wangxin) [inline tests/dualtor/test_server_failure.py:18]: The condition removed here need to be added to the `tests_mark_conditions.yaml` file.

File: tests/dualtor/test_server_failure.py [hunk 3]
```diff
@@ -51,7 +47,11 @@ def lower_tor_mux_state_verfication(state, health):
     pytest_assert(wait_until(20, 1, 0, upper_tor_mux_state_verification, 'active', 'unhealthy'),
                   "mux_cable status is unexpected. Should be (active, unhealthy)")
     # Verify mux_cable state on lower_tor is standby
-    pytest_assert(wait_until(20, 1, 0, lower_tor_mux_state_verfication, 'standby', 'unhealthy'),
+    if is_cisco_device(duthost):
+      pytest_assert(wait_until(30, 1, 0, lower_tor_mux_state_verfication, 'standby', 'unhealthy'),
```
Comment (kevinskwang) [inline tests/dualtor/test_server_failure.py:51]: No need separate different value for Cisco

**Comment by svelamal** [comment]
New PR raised without making the pytest mark changes

**Pull request closed without merging**